src/Messaging/Messaging.ts

```diff
@@ -101,4 +101,8 @@ export class Messaging {
       consume(claimIPFSHashPairs)
     })
   }
+
+  publishClaimsNotDownloaded = async (IPFSHashFailure: ReadonlyArray<IPFSHashFailure>) => {
```

Comment:
please make this value `ipfsHashFailure` so that it's not the same as the interface.